package.json

```diff
@@ -14,22 +14,18 @@
   },
   "homepage": "https://github.com/scality/vaultclient#readme",
   "dependencies": {
-    "@aws-crypto/sha256-universal": "^2.0.1",
-    "@smithy/signature-v4": "^2.1.1",
-    "commander": "2.20.0",
-    "httpagent": "git+https://github.com/scality/httpagent#1.0.6",
-    "werelogs": "scality/werelogs#8.1.3",
-    "xml2js": "0.4.19"
+    "@aws-crypto/sha256-universal": "^5.2.0",
+    "@smithy/signature-v4": "^4.1.0",
+    "commander": "^11.0.0",
+    "httpagent": "git+https://github.com/scality/httpagent#99e159de4763134f9369c14c32c6eac0689d2b13",
+    "werelogs": "scality/werelogs#0ab4ca3470a2f03f34d33fdc2689fda414f7c206",
```

Comment:
shouldnt this be changed to a proper version tag ?

Reply:
Yes absolutely, I still have this pr open so I'll change before merging